### PR TITLE
fix tmp dir saving

### PR DESCRIPTION
**Pull request by eaidova**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: tests/ipex/test_modeling.py [hunk 1]
```diff
@@ -44,6 +43,7 @@
     IPEXModelForSequenceClassification,
     IPEXModelForTokenClassification,
 )
+from optimum.intel.openvino.utils import TemporaryDirectory
```
Comment (IlyasMoutawwakil) [inline tests/ipex/test_modeling.py:46]: you're importing optimum.intel.openvino.utils in ipex paths, no need to change anything in ipex paths if I understand correctly.

**Review by IlyasMoutawwakil — CHANGES_REQUESTED** [review]
ipex tests are failing, please make sure tests are passing before requesting a review.

File: tests/ipex/test_modeling.py (hunk 1 above)
Comment (eaidova) [inline tests/ipex/test_modeling.py:46]:
```suggestion
```

Comment (eaidova) [reply tests/ipex/test_modeling.py]: it was commited accidentally, sorry

**Review by IlyasMoutawwakil — APPROVED** [review]

**Pull request merged**